docs/podman.1.md

```diff
@@ -192,7 +192,7 @@ the exit codes follow the `chroot` standard, see below:
     When Podman runs in rootless mode, the file `$HOME/.config/containers/storage.conf` is also loaded.
 
 ## Rootless mode
-Podman can also be used as non-root user.  When podman runs in rootless mode, an user namespace is automatically created.
+Podman can also be used as non-root user.  When podman runs in rootless mode, the users user namespace is automatically created, defined in /etc/subuid and /etc/subgid.
```

Comment:
suggest: "the users user namespace is automatically created" to "a user namespace is automatically created for the user"

Reply:
Fixed